bin/check_samplesheet.py

```diff
@@ -86,11 +89,10 @@
 
             ## CHECK GENOME EXTENSION
             if len(genome.split('.')) > 1:
-                if genome[-6:] != '.fasta' and genome[-3:] != '.fa':
-                    print "{}: Genome field incorrect extension (has to be '.fasta' or 'fa')!\nLine: '{}'".format(ERROR_STR,line.strip())
+                if genome[-9:] != '.fasta.gz' and genome[-6:] != '.fa.gz':
```

Comment:
```suggestion
                if genome[-6:] != '.fasta' and genome[-3:] != '.fa' and genome[-9:] != '.fasta.gz' and genome[-6:] != '.fa.gz':
```

Reply:
We still need to allow for `.fasta` and and `.fa` extensions because most people will have them in this format. All of the `iGenomes` files are in this format too. I think this is why the Travis CI tests are failing because the test-dataset has a `.fa` extension. We could update that to be zipped after we have merged this :+1:

Reply:
Could simplify that using a `for` loop?
```
for ext in ['.fasta', '.fa', '.fasta.gz', '.fa.gz']:
    if genome[-len(ext):] != ext:
```